lib/filter/tests/test_filters_in_list.c

```diff
@@ -23,8 +23,8 @@
  */
 
 #include <stdlib.h>
-#include <glib.h>
 
+#include "compat/glib.h"
 #include "cfg.h"
```

Comment:
syslog-ng.h already includes glib.h, so we shouldn't include glib directly, but we should include syslog-ng.h as the first thing except if there's already a header that does that.

Reply:
Oh okay, I'll replace them.
Thanks, @bazsi.

Reply:
I just leave this here as a note:

In my opinion, the way we use `GLIB_DISABLE_DEPRECATION_WARNINGS` is not sustainable, I find it risky.
We should make it at least independent from the include order of `compat/glib.h`.

Using `-DGLIB_DISABLE_DEPRECATION_WARNINGS` as a CPPFLAG would be an alternative.